sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ReadmeSamples.java

```diff
@@ -341,7 +348,153 @@ public void databaseCreateContainerSample() {
+    }
+
+    public void createGlobalThroughputControlConfigBuilderSample() {
+        CosmosAsyncClient cosmosAsyncClient = new CosmosClientBuilder()
+            .buildAsyncClient();
+        // BEGIN: com.azure.cosmos.CosmosAsyncClient.createGlobalThroughputControlConfigBuilder
+        String databaseId = "<YOUR DATABASE NAME>";
+        String containerId = "<YOUR CONTAINER NAME>";
+
+        GlobalThroughputControlConfig config =
+            cosmosAsyncClient.createGlobalThroughputControlConfigBuilder(databaseId, containerId)
+                .build();
+        // END: com.azure.cosmos.CosmosAsyncClient.createGlobalThroughputControlConfigBuilder
+    }
+
+    public void readConflictSample() {
+        // BEGIN: com.azure.cosmos.CosmosAsyncConflict.read
+        String conflictId = "CONFLICT_ID";
+        CosmosAsyncDatabase cosmosAsyncDatabase =
+            new CosmosAsyncDatabase(conflictId, cosmosAsyncClient);
+        CosmosAsyncContainer cosmosAsyncContainer =
+            new CosmosAsyncContainer(conflictId, cosmosAsyncDatabase);
+        CosmosConflictRequestOptions options =
+            new CosmosConflictRequestOptions();
+        CosmosAsyncConflict conflict =
+            new CosmosAsyncConflict(conflictId, cosmosAsyncContainer);
+
+        Mono<CosmosConflictResponse> conflictResponseMono = conflict.read(options);
+        // END: com.azure.cosmos.CosmosAsyncConflict.read
+    }
+
+    public void CosmosAsyncProcedureReadSample() {
+        CosmosAsyncDatabase cosmosAsyncDatabase = new CosmosAsyncDatabase(id, cosmosAsyncClient);
+        // BEGIN: com.azure.cosmos.CosmosAsyncStoredProcedure.read
+        CosmosAsyncContainer cosmosAsyncContainer =
+            new CosmosAsyncContainer(id, cosmosAsyncDatabase);
+        CosmosAsyncStoredProcedure procedure =
+            new CosmosAsyncStoredProcedure(id, cosmosAsyncContainer);
+
+        Mono<CosmosStoredProcedureResponse> response = procedure.read();
+        // END: com.azure.cosmos.CosmosAsyncStoredProcedure.read
+    }
+
+    public void CosmosAsyncProcedureDeleteSample() {
+        CosmosAsyncDatabase cosmosAsyncDatabase = new CosmosAsyncDatabase(id, cosmosAsyncClient);
+        CosmosAsyncContainer cosmosAsyncContainer =
+            new CosmosAsyncContainer(id, cosmosAsyncDatabase);
+        // BEGIN: com.azure.cosmos.CosmosAsyncStoredProcedure.delete
+        CosmosAsyncStoredProcedure procedure =
+            new CosmosAsyncStoredProcedure(id, cosmosAsyncContainer);
+        CosmosStoredProcedureRequestOptions options =
+            new CosmosStoredProcedureRequestOptions();
+
+        Mono<CosmosStoredProcedureResponse> response = procedure.delete();
+        // END: com.azure.cosmos.CosmosAsyncStoredProcedure.delete
+    }
+
+    public void CosmosAsyncProcedureExecuteSample() {
+        // BEGIN: com.azure.cosmos.CosmosAsyncStoredProcedure.execute
+        String id = "ID";
+        CosmosAsyncStoredProcedure procedure =
```

Comment:
Below test files can point you to the right direction: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/StoredProcedureCrudTest.java